airbyte-integrations/connectors/destination-elasticsearch-strict-encrypt/README.md

```diff
@@ -0,0 +1,5 @@
+# Elasticsearch Strict Encrypt Test Configuration
```

Comment:
could you please update the readme file according to the certification requirements?